### PR TITLE
updated SparkConfView in views.py and couture_config.html

### PR DESCRIPTION
**Pull request by AdrianMirandaTailor**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [ ] Description above provides context of the change
- [ ] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [ ] Unit tests coverage for changes (not needed for documentation changes)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions.
- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/config_templates/default_airflow.cfg [hunk 1]
```diff
@@ -102,7 +102,7 @@ executor = SequentialExecutor
 # The SqlAlchemy connection string to the metadata database.
 # SqlAlchemy supports many different database engine, more information
 # their website
-sql_alchemy_conn = sqlite:///{AIRFLOW_HOME}/airflow.db
+sql_alchemy_conn = mysql:///{AIRFLOW_HOME}/airflow.db
```
Comment (Anubhav2701) [inline airflow/config_templates/default_airflow.cfg]: No need for this change

File: airflow/config_templates/default_airflow.cfg [hunk 2]
```diff
@@ -413,7 +413,7 @@ hide_paused_dags_by_default = False
 page_size = 100
 
 # Use FAB-based webserver with RBAC feature
-rbac = False
+rbac = True
```
Comment (Anubhav2701) [inline airflow/config_templates/default_airflow.cfg]: NO need for this change.

File: airflow/config_templates/default_airflow.cfg [hunk 3]
```diff
@@ -1069,3 +1069,7 @@ reconnect_interval_in_ms = 30000
 pytorch_model_server_host = pytorchserving
 pytorch_model_server_management_port = 8060
 pytorch_model_server_inference_port = 8601
+
+
+SQLALCHEMY_DATABASE_URI = "mysql://localhost:3306/airflowDB"
```
Comment (Anubhav2701) [inline airflow/config_templates/default_airflow.cfg]: no need for this change

File: airflow/config_templates/default_airflow.cfg [hunk 4]
```diff
@@ -1069,3 +1069,7 @@ reconnect_interval_in_ms = 30000
...
+SQLALCHEMY_BINDS = ""
```
Comment (Anubhav2701) [inline airflow/config_templates/default_airflow.cfg]: no need for this change

File: airflow/config_templates/default_airflow.cfg [hunk 5]
```diff
@@ -1069,3 +1069,5 @@ reconnect_interval_in_ms = 30000
 pytorch_model_server_host = pytorchserving
 pytorch_model_server_management_port = 8060
 pytorch_model_server_inference_port = 8601
+
+
```
Comment (AnubhavUjjawal) [inline airflow/config_templates/default_airflow.cfg]: Remove these lines.

File: airflow/www_rbac/templates/airflow/mod1.html [hunk 6]
```diff
@@ -0,0 +1,600 @@
...
+// getAllNames({{Files | safe}},'{{Arguments['jars']}}');
+// showcouturecheckboxes(true);
+
+// Py_getAllNames({{Py_Files | safe}},'{{Arguments['py-files']}}');
+//  Py_showcouturecheckboxes(true);
+
+// kt_getAllNames({{kt_Files | safe}},'{{Arguments['keytab']}}');
+// kt_showcouturecheckboxes(true);
+//   }
+  </script>
```
Comment (AnubhavUjjawal) [inline airflow/www_rbac/templates/airflow/mod1.html]: Do we need this file ?

File: airflow/www_rbac/templates/airflow/couture_config.html [hunk 7]
```diff
@@ -398,186 +320,118 @@ <h3>Configurations</h3>
...
-            for (var i=0;i<selected.length;i++)
-            {
-                if(file_name == selected[i]) {
-                    in_selected= true;
-                }
-            }
-            if (in_selected==true) {
-                kt_checkBox.checked = "true";
+//function that gets the names of files and puts it onto the required labels
+function getAllNames_test(files,selected_files,num) {
```
Comment (AnubhavUjjawal) [inline airflow/www_rbac/templates/airflow/couture_config.html]: change the function name. Also, either use camelCase or snake_case. Don't use both. For function names, use camelCase.

File: airflow/www_rbac/templates/airflow/couture_config.html [hunk 8]
```diff
@@ -322,64 +239,69 @@ <h3>Configurations</h3>
...
-    document.getElementById('container2').appendChild(div1);
+    div1.innerHTML = document.getElementById(update).innerHTML;
+    document.getElementById(containerNo).appendChild(div1);
   }
 }
+
 var expanded = false;
-count_of_clicks=1
+count_of_clicks=1;
+var Py_expanded = false;
```
Comment (AnubhavUjjawal) [inline airflow/www_rbac/templates/airflow/couture_config.html]: change the function name. Also, either use camelCase or snake_case. Don't use both.Use camelCase.

Comment (AnubhavUjjawal) [reply airflow/www_rbac/templates/airflow/couture_config.html]: Change the naming everywhere where it is not proper.

File: airflow/www_rbac/views.py [hunk 9]
```diff
@@ -3557,155 +3559,121 @@ def dashboard_view(self, filename):
...
     @action_logging
+    
+
+    #function that updates couture_conf.html page and writes changes to couture-spark config file 
     def update_spark_conf(self, group):
-        # print(request.form)
         title = "Couture Spark Configuration"
-        import collections
-        import configparser as CP
+        default_view = 'update_spark_conf'
```
Comment (AnubhavUjjawal) [inline airflow/www_rbac/views.py]: I think this should be at L3562.

File: airflow/www_rbac/views.py [hunk 10]
```diff
@@ -3556,156 +3558,153 @@ def dashboard_view(self, filename):
...
+            for file in f:
+                if ext=="kt":
+                    files.append(file)
+                else:
+                    if file.endswith(ext):
+                        files.append(file)
+                    if ext == ".py" and (
+                        file.endswith(".egg") or file.endswith(".zip")):
+                        files.append(file)
+        return files
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: Think of how you can make this better.

File: airflow/www_rbac/views.py [hunk 11]
```diff
@@ -3556,156 +3558,147 @@ def dashboard_view(self, filename):
         return self.render_template('eda/eda_outputs.html', visualisations=viz)
 
 
+ 
 class SparkConfView(AirflowBaseView):
-    default_view = 'update_spark_conf'
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py:3560]: Where is this line.

**Pull request closed without merging**